xs2a-gateway-service-api-adapter/src/main/java/de/adorsys/xs2a/gateway/adapter/AbstractService.java

```diff
@@ -25,16 +25,22 @@
 import de.adorsys.xs2a.gateway.service.exception.ErrorResponseException;
 import de.adorsys.xs2a.gateway.service.exception.NotAcceptableException;
 
+import java.io.ByteArrayOutputStream;
 import java.io.IOException;
 import java.io.PushbackInputStream;
 import java.io.UncheckedIOException;
+import java.nio.charset.StandardCharsets;
 import java.util.Map;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
 
 public abstract class AbstractService {
```

Comment:
I propose to add logger and log all exception and all necessary actions

Reply:
Done, thanks. Please, take a look, whether it is done as you expected.

Reply:
Please rename LOG variable to lower case

Reply:
Done, thanks!